tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -96,6 +99,16 @@ public function testNoMethodApiLoader()
         $routeCollection = $this->getApiLoaderWithResourceMetadata($resourceMetadata)->load(null);
     }
 
+    /**
+     * @expectedException \ApiPlatform\Core\Exception\InvalidResourceException
+     */
+    public function testNoShortNameApiLoader()
+    {
+        $resourceMetadata = new ResourceMetadata();
```

Comment:
What do you mean?  
`$this->getApiLoaderWithResourceMetadata(new ResourceMetadata())->load(null);` ?